docs/guide/config_back_office.md

```diff
@@ -147,6 +147,6 @@ ezpublish:
                    - 43 # Media
 ```
 
-## Universal Discovery module limits
+## Universal Discovery Widget (UDW) configuration
 
-Limits for different sections of Universal Discovery Widget module (UDW) can be defined through respective settings in [`universal_discovery_widget.yml`](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/config/universal_discovery_widget.yml)
\ No newline at end of file
+The Universal Discovery Widget module (UDW) can be found in [Extending modules](extending_modules.md#universal-discovery-widget-module)
```

Comment:
The link will be broken (heading changed)

Reply:
Good catch, it worked when I checked because it was first heading on the page